lib/api/controllers/funnelController.js

```diff
@@ -365,7 +380,9 @@ class FunnelController {
 
     let modifiedRequest = request;
 
-    return this.kuzzle.pipe(this.getEventName(request, 'before'), request)
+    return Bluebird.resolve(request)
```

Comment:
You can replace this call to `Bluebird.resolve` by `this.performDocumentAlias(newRequest, 'before')`